src/sys/socket/addr.rs

```diff
@@ -975,10 +975,22 @@ pub trait SockaddrLike: private::SockaddrLikePriv {
     /// Some C APIs from provide `len`, and others do not.  If it's provided it
     /// will be validated.  If not, it will be guessed based on the family.
     ///
+    /// # Arguments
+    ///
+    /// - `addr`:   raw pointer to something that can be cast to a
+    ///             `libc::sockaddr`. For example, `libc::sockaddr_in`,
+    ///             `libc::sockaddr_in`, etc.
```

Comment:
Should the second `libc::sockaddr_in` be `libc::sockaddr_in6`?

Reply:
Yep.